src/expressions.rs

```diff
@@ -23,10 +26,19 @@ pub(crate) fn chat_display_name(chat_expression: &gtk::Expression) -> gtk::Expre
 pub(crate) fn user_full_name(user_expression: &gtk::Expression) -> gtk::Expression {
```

Comment:
While I find these changes ok, I suggest renaming this function to something like `user_display_name`, which is more meaningful of what it does now. Together with that, the documentation above should be updated.